cmd/airgap/listimages.go

```diff
@@ -33,10 +34,12 @@ func NewAirgapListImagesCmd() *cobra.Command {
 		Short:   "List image names and version needed for air-gap install",
 		Example: `k0s airgap list-images`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			// we don't need warning messages in case of default config
-			logrus.SetLevel(logrus.ErrorLevel)
 			c := CmdOpts(config.GetCmdOpts())
-			cfg, err := config.GetYamlFromFile(c.CfgFile, c.K0sVars)
+			// we don't need warning messages in case of default config
+			c.Logger = util.CLILogger()
```

Comment:
Setting `c.Logger = util.CLILogger()` in `RunE` seems inconsistent in these changes (intentionally?).  I see that `listimages.go` sets it, but others like `etcd.go` don't.  Is this intentional?

Reply:
CLILogger is only used when the command is a distinctive CLI command (unlike controller, api and worker commands that will normally need a more detailed log output, for example).
As for this specific use case (etcd command, which is a CLI command) I updated the PR.